bot/utils/messages.py

```diff
@@ -98,7 +98,14 @@ async def wait_for_deletion(
         try:
             await bot.instance.wait_for('reaction_add', check=check, timeout=timeout)
         except asyncio.TimeoutError:
-            await message.clear_reactions()
+            try:
```

Comment:
Can't this be simplified into the following to remove these nested try except blocks ?
```suggestion
try:
	await bot.instance.wait_for('reaction_add', check=check, timeout=timeout)
except asyncio.TimeoutError:
	try:
		await message.clear_reactions()
	except discord.HTTPException as e:
		if not isinstance(message.channel, discord.Thread):
			# Threads might not be accessible by the time we try to remove the reaction.
			raise e

try:
	await message.delete()
except discord.NotFound:
    log.trace(f"wait_for_deletion: message {message.id} deleted prematurely.")
```

Reply:
28574243